tasks/eth/base-002-fp-upgrade/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,296 @@
+    using LibString for string;
+
+    // Safe contract for this task.
+    GnosisSafe securityCouncilSafe = GnosisSafe(payable(vm.envAddress("COUNCIL_SAFE")));
+    GnosisSafe fndSafe = GnosisSafe(payable(vm.envAddress("FOUNDATION_SAFE")));
+    GnosisSafe ownerSafe = GnosisSafe(payable(vm.envAddress("OWNER_SAFE")));
+
+    /// @notice Verify against https://docs.optimism.io/chain/security/privileged-roles#guardian
+    address constant superchainConfigGuardian = 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A;
+    
+    address constant optimismPortalGuardian = 0x09f7150D8c019BeF34450d6920f6B3608ceFdAf2;
+
+    /// @notice Verify against https://github.com/ethereum-optimism/superchain-registry/blob/21506ecedf6e83410d12c7cc406685ac061a2a74/superchain/configs/mainnet/base.toml#L44
+    address constant p2pSequencerAddress = 0xAf6E19BE0F9cE7f8afd49a1824851023A8249e8a;
+
+    /// @notice Verify against https://github.com/ethereum-optimism/superchain-registry/blob/21506ecedf6e83410d12c7cc406685ac061a2a74/superchain/configs/mainnet/base.toml#L9
+    address constant batchInboxAddress = 0xFf00000000000000000000000000000000008453;
+
+    /// @notice Verify against https://github.com/ethereum-optimism/superchain-registry/blob/21506ecedf6e83410d12c7cc406685ac061a2a74/superchain/configs/mainnet/base.toml#L33
+    address constant batchSenderAddress = 0x5050F69a9786F081509234F1a7F4684b5E5b76C9;
+
+    /// @notice Verify against https://github.com/ethereum-optimism/superchain-registry/blob/21506ecedf6e83410d12c7cc406685ac061a2a74/superchain/configs/mainnet/base.toml#L39
+    address constant systemConfigOwner = 0x14536667Cd30e52C0b458BaACcB9faDA7046E056;
+
+    /// @notice Verify onchain
+    uint256 constant l2GenesisBlockGasLimit = 0xABA9500;
+
+    /// @notice Verify onchain
+    uint256 constant gasPriceOracleOverhead = 0;
+
+    /// @notice Verify with `cast call 0x73a79Fab69143498Ed3712e519A88a918e1f4072 "scalar()(bytes32)" --rpc-url https://ethereum-rpc.publicnode.com`. We don't link to
+    /// the deploy-config here because it needs to be updated to account for the new way the scalar
+    /// is encoded post-ecotone. But we don't want this parameter to change, so we hardcode the
+    /// existing value (fetched with the above command) here.
+    uint256 constant gasPriceOracleScalar = 0x01000000000000000000000000000000000000000000000000101c12000008dd;
+
+    /// @notice Verify against
+    uint256 constant systemConfigStartBlock = 17482144;
+
+    // Verify against the `DisputeGameFactoryProxy` deployment address
+    address constant dgfProxy = 0x43edB88C4B80fDD2AdFF2412A7BebF9dF42cB40e;
+
+    address councilSafe;
+    address foundationSafe;
+
+    Types.ContractSet proxies;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        councilSafe = vm.envAddress("COUNCIL_SAFE");
+        foundationSafe = vm.envAddress("FOUNDATION_SAFE");
```

Comment:
We have both these these vars, and `securityCouncilSafe` and `fndSafe`. Let's dedupe them, and we should rename the council safe solidity variable (not the env var) to `baseSafe`